### DISCUSSION TITLE
Understanding models names - Carryover, Ad-stock and Hill-adstock

### DISCUSSION THREAD

**Discussion by AruunnAnanda** [google/lightweight_mmm#128] [Q&A]
Hi there,

Are there any example graphs for these models names to visually understand the transformation? It's kind of hard to directly visualise from the equation. 

And also, can you please elaborate these models by writing a few lines about them? 

Thank you!

**Comment by statm3n** [comment]
Hi, @AruunnAnanda,

These are great questions that will help to understand MMM model.

1. check the [media transformations](https://github.com/google/lightweight_mmm/blob/5abe5bae0e91852d7fa2a3fc0618f66d22adee1d/lightweight_mmm/media_transforms.py#L63) starting from line 63 and visualize them as you wanted
2. you could find the one-liner for these 3 models in [readme](https://github.com/google/lightweight_mmm/tree/5abe5bae0e91852d7fa2a3fc0618f66d22adee1d#media-saturation-and-lagging)

Cheers!
